### DISCUSSION TITLE
U plan to share your hardware design?

### DISCUSSION THREAD

**Discussion by CIRCUITF0RGE** [eh2k/squares-and-circles#115] [Q&A]

**Comment by eh2k** [comment]
I have designed several RevB variants that have not yet been built. The first version as seen [here](https://github.com/eh2k/squares-and-circles/wiki/Squares&Circles-D_SM-Prototype) works as expected. Maybe I have one plain PCB+panel left (If you are interested in building it, just send me an email for details).

The RevB I would like to discuss in a group of RevA builders/users. And if there is a RevB, almost everything (daisy patch sm and other chips) from the RevA can be reused there.

I may end up publishing just the hardware design, and/or the PCBs may be available somewhere.

**Comment by CIRCUITF0RGE** [comment]
Hey,

yeah cool would be interested to build one. I need a Teensy 4.0 if i got it
right? can make the pcbs here since i literally live where all the FAB
HOUSES are lol....think its cheaper then send a set....

Am So., 29. Okt. 2023 um 16:30 Uhr schrieb eh2k ***@***.***>:
